contrib/ray/raycluster_example.yaml

```diff
@@ -1,60 +1,112 @@
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-ray-workers-head
+spec:
+  action: ALLOW
+  rules:
+  - to:
+    - operation:
+        ports:
+        - "6379"
```

Comment:
Please make sure that you only allow traffic from the default-editor serviceaccount (the one that the workers use as well) in the same namespace